.github/workflows/on_release_publish.yml

```diff
@@ -39,6 +39,18 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.TOKEN_BOT }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      # Chromatic
+      - name: Build Storybook
+        run: yarn build-storybook
+
+      - name: Run Chromatic
```

Comment:
I wonder if this should be its own job. Do we need all the previous steps to run chromatic?